### PR TITLE
Damage Number Integration

### PR DESCRIPTION
**Pull request by Surilexa**
Prefab Numbers.
Damage numbers spawn to reflect the boss taking damage.
Immune is shown with a 1 second cooldown on every hit against the boss shield.
Numbers are either shown as red or orange, and have a range of text size.
Boss now has an empty Game object for number source location.

### CODE REVIEW

File: Assets/Scripts/Gamemode/LoadGameAssets.cs
Comment (jss333) [inline Assets/Scripts/Gamemode/LoadGameAssets.cs:1]: Is this script (and corresponding object in the scene) doing anything? If not, please delete it.

File: Assets/Scripts/Enemy/Boss/BossController.cs [hunk 1]
```diff
@@ -28,6 +34,8 @@ public class BossController : MonoBehaviour
     public AudioClip damageTakenSFX;
     public float damageTakenSFXCooldown = 0.2f;
     private float lastDamageTakenSFXPlayTime = -Mathf.Infinity;
+    public float stopOverflowDamageNumbers = 2f;
+    private float ph;
```
Comment (jss333) [inline Assets/Scripts/Enemy/Boss/BossController.cs]: What does `ph` stand for? Please use a more descriptive name.

**Review by jss333 — CHANGES_REQUESTED** [review]
Hi @Surilexa the functionality works well, but I've refactored the code to simplify it and to follow the style guide. Please review my commit and the changes I've made and let me know if you have any comments.

Additionally, please address the comments I've left on the PR, thanks.

File: Assets/Scripts/Enemy/Boss/BossController.cs (hunk 1 above)
Comment (Surilexa) [reply Assets/Scripts/Enemy/Boss/BossController.cs]: it was a literal Place Holder name that I didnt change

File: Assets/Scripts/Gamemode/LoadGameAssets.cs
Comment (Surilexa) [reply Assets/Scripts/Gamemode/LoadGameAssets.cs:1]: Deleted

**Review by jss333 — APPROVED** [review]
LGTM

**Pull request merged**